mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -104,6 +107,23 @@ var (
 	}
 
 <% end -%>
+
+<% unless version == "ga" -%>
```

Comment:
```suggestion
```
we can remove these lines

Reply:
I agree, but I believe it would be easier in future to move features to GA if each one is surrounded with its own unless-end block